graphene_django/static/graphene_django/graphiql.js

```diff
@@ -173,6 +175,7 @@
     onEditQuery: onEditQuery,
     onEditVariables: onEditVariables,
     onEditOperationName: onEditOperationName,
+    headerEditorEnabled: GRAPHENE_SETTINGS.graphiqlHeaderEditorEnabled || true,
```

Comment:
```suggestion
    headerEditorEnabled: GRAPHENE_SETTINGS.graphiqlHeaderEditorEnabled,
```

Otherwise setting `graphiqlHeaderEditorEnabled` to `false` will still set the option to `true`.